### ISSUE TITLE
RecoverVessel doesn't appear to be recognized in Space Center view

### ISSUE THREAD

**Issue by inigmatus** [jrossignol/ContractConfigurator#343]
Attempted to recover landed vessel using RecoverVessel ... but only after exiting to Space Center. Vessel could be recovered, but recovery was not recognized by Contract.

**Comment by jrossignol** [comment]
This one is unfixable without re-writing large amounts of code that rely on Vessel rather than vessel id or ProtoVessel.  I'll have to think about what to do with RecoverVessel, but for now I'd recommend using ReturnHome over RecoverVessel for most scenarios.

**Comment by inigmatus** [comment]
I will need to test, but I think RecoverKerbal is affected by this as well.  With passenger contracts, the intent is to recover returning passengers to complete the contract (so players can't abuse them after contract completion).

**Comment by jrossignol** [comment]
RecoverKerbal should not be affected - please raise a separate issue if you
think there is a problem there.

**Comment by inigmatus** [comment]
RecoverKerbal works perfect.  Recoding my contracts now for RecoverKerbal as I consider RecoverVessel bugged (players will invariably attempt recovery of aircraft from Space Center View). You might want to put a note on this limitation in the wiki for now so modders are given a heads up.
